File: tests/testdata/codegen/simple-splat-pp/nodejs/simple-splat.ts
import * as khulnasoft from "@khulnasoft/khulnasoft";
import * as splat from "@khulnasoft/splat";

const allKeys = splat.getSshKeys({});
const main = new splat.Server("main", {sshKeys: allKeys.then(allKeys => allKeys.sshKeys.map(__item => __item.name))});

File: tests/testdata/codegen/simple-splat-pp/go/simple-splat.go
package main

import (
	"example.com/khulnasoft-splat/sdk/go/splat"
	"github.com/khulnasoft/khulnasoft/sdk/v3/go/khulnasoft"
)

func main() {
	khulnasoft.Run(func(ctx *khulnasoft.Context) error {
		allKeys, err := splat.GetSshKeys(ctx, map[string]interface{}{}, nil)
		if err != nil {
			return err
		}
		var splat0 []string
		for _, val0 := range allKeys.SshKeys {
			splat0 = append(splat0, val0.Name)
		}
		_, err = splat.NewServer(ctx, "main", &splat.ServerArgs{
			SshKeys: splat0,
		})
		if err != nil {
			return err
		}
		return nil
	})
}
